src/js/redux/reducers/explorer/explorerReducer.js

```diff
@@ -14,12 +14,12 @@ export const ActiveScreen = new Record({
 });
 
 const initialQuarters = defaultQuarters();
-
+const initalPeriods = defaultPeriod();
```

Comment:
We should be able to just use [this fn in the bulkDownloadReducer](https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/redux/reducers/bulkDownload/bulkDownloadReducer.js#L14) for this.

Reply:
We can't, the `getInitialPeriodByFy` function gets the latest period in the lastest closed quarter, which is not the latest closed period if that makes sense. Essentially the period will close before the quarter will, but that function is checking for closed quarters, which is what was causing the problem